src/components/ImageSlider/ImageSlider.js

```diff
@@ -0,0 +1,29 @@
+import './ImageSlider.css';
+import {results as data} from '../../utils/mocks/en-us/featured-banners.json';
+import { FaArrowAltCircleRight, FaArrowAltCircleLeft } from 'react-icons/fa';
+
+const ImageSlider = () => {
+  console.log(data);
+  return (
+    <section className={'slider'}>
+      <FaArrowAltCircleLeft className='left-arrow'  />
+      <FaArrowAltCircleRight className='right-arrow'  />
+      {
+        data.map((slide, index) => {
+          return (
+            <div
+              className={index === 0 ? 'slide active' : 'slide'}
+              key={index}
+            >
+              {index === 0 && (
+                <img src={slide.data.main_image.url} alt={'test'} className='image' />
```

Comment:
Let's keep the json structure implementation detail out of the `ImageSlider` component. We can, for example, receive in the props an array of slides that have the properties description, imageSource and title and let another component or hook extract these values from the json

Reply:
I separated the JSON from the component and now it's received through props, I have pending separate the map in the render function, I added a TODO